manim/__main__.py

```diff
@@ -72,10 +73,15 @@ def main():
 
         # elif args.cmd == "some_other_cmd":
         #     something_else_here()
+        # TODO: Maybe make use of this when I understand this?
```

Comment:
Nitpick: please remove all extraneous lines and comments before this is merged.

Reply:
I'd have to pretend that that wasn't an actual reminder to myself

But I've half forgotten why I did that so that makes sense